generate/src/contract/deployment.rs

```diff
@@ -175,12 +171,7 @@ fn expand_deploy(cx: &Context) -> Result<TokenStream> {
             }
 
             fn at_address(web3: #ethcontract::DynWeb3, address: #ethcontract::Address, _: Self::Context) -> Self {
-                use #ethcontract::Instance;
-
-                let abi = Self::artifact().abi.clone();
-                Self {
-                    instance: Instance::at(web3, abi, address),
-                }
+                Self::at(&web3, address)
```

Comment:
This method existed before, right? It's not added in this PR

Reply:
It changed because of how we achieve `Deref<Target = Methods>`.